### ISSUE TITLE
No archive entry found for components ESP on anlworkstation

### ISSUE THREAD

**Issue by dqwu** [ESMCI/cime#2660]
After PR #2656 (Archive test refinement) was merged, a single
cime_developer test (e.g. SMS.T42_T42.S) fails during run.
```
Exception during run:
ERROR: No archive entry found for components: ['ESP']
Traceback (most recent call last):
  File "scripts/lib/CIME/SystemTests/system_tests_common.py", line 155, in run
    self._st_archive_case_test()
  File "scripts/lib/CIME/SystemTests/system_tests_common.py", line 286, in _st_archive_case_test
    result = self._case.test_env_archive()
  File "scripts/lib/CIME/case/case_st_archive.py", line 749, in test_env_archive
    expect(not components, "No archive entry found for components: {}".format(components))
  File "scripts/lib/CIME/utils.py", line 130, in expect
    raise exc_type(msg)
SystemExit: ERROR: No archive entry found for components: ['ESP']
```
If we remove the following line in system_tests_common.py the test can pass:
`self._st_archive_case_test()`

How to define archive entry for ESP on anlworkstation?

**Comment by jedwards4b** [comment]
E3sm Should not look for an esp entry - I will fix this.

**Comment by dqwu** [comment]
@jedwards4b 
This issue (2660) has been fixed by PR 2663 (verified on anlworkstation)
However, it is not closed as PR 2663 uses a wrong issue number (2659)
Could you please modify PR 2663 to use issue number 2660?

**Comment by jedwards4b** [comment]
Fixed by #2663
